src/pytorch_lightning/strategies/single_tpu.py

```diff
@@ -70,9 +67,6 @@ def setup(self, trainer: "pl.Trainer") -> None:
         self.tpu_local_core_rank = xm.get_local_ordinal()
         self.tpu_global_core_rank = xm.get_ordinal()
 
-    def model_to_device(self) -> None:
-        self.model.to(self.root_device)
```

Comment:
why remove this?

Reply:
Because it looked like code duplication since the function is the exact same as the one in the parent class ```SingleDeviceStrategy```. 
I noticed it because the mypy error is the same as the one in the parent class which is fixed in https://github.com/Lightning-AI/lightning/pull/13532.
Do I have to put it back?

Reply:
Either way is fine with me but keep in mind we would prefer focus on fixing the typing errors and not mix different types of refactors :) Otherwise LGTM